docs/root/intro/arch_overview/load_balancing.rst

```diff
@@ -94,10 +94,74 @@ Priority levels
 ------------------
 
 During load balancing, Envoy will generally only consider hosts configured at the highest priority
-level. For each EDS :ref:`LocalityLbEndpoints<envoy_api_msg_LocalityLbEndpoints>` an optional priority
-may also be specified. Currently the failover mechanics for routing from one priority level to
-another are fairly simplistic: a given priority level will be used until it has zero healthy hosts,
-at which point it will hard fail to the next highest priority level.
+level. For each EDS :ref:`LocalityLbEndpoints<envoy_api_msg_LocalityLbEndpoints>` an optional
+priority may also be specifie. When endpoints at the highest priority level (P=0) are healthy, all
+traffic will land on endpoints in that priority leve. As endpoints for the highest priority level
+become unhealthy, traffic will begin to trickle to lower priority levels.
+
+Currently, it is assumed that each priority level is over-provisioned by a (hard-coded) factor of
+1.4. So if 80% of the endpoints are healthy, the priority level is still considered healthy because
+80*1.4 > 10. As the number of healthy endpoints dips below 72%, the health of the priority level
+goes below 100, and any residual traffic will flow to the next priority level.
+
+Assume a simple set-up with 2 priority levels, P=1 100% healthy.
+
++----------------------------+---------------------------+----------------------------+
+| Percent healthy endpoints  | Percent of traffic to P=0 |  Percent of traffic to P=1 |
++============================+===========================+============================+
+| 100                        | 100                       |   0                        |
++----------------------------+---------------------------+----------------------------+
+| 72                         | 100                       |   0                        |
++----------------------------+---------------------------+----------------------------+
+| 71                         | 99                        |   1                        |
++----------------------------+---------------------------+----------------------------+
+| 50                         | 70                        |   30                       |
```

Comment:
I think it would be helpful to see the expression that is used to compute the residue here.

Reply:
I used the expressions below to try and compute the 50% health case, I get P_0 traffic at 33% and P_1 traffic at 67%. Maybe I did math wrong, can you double check?

Reply:
so percent healthy (healthy_PX_backends / total_PX_backends) is 50%
health(P_X) = (140 * .5) = 70
P_0 = min(70, 100) = 70
load to P_1= 100 - (70) = 30